src/Pages/Home/Home.js

```diff
@@ -1,20 +1,32 @@
+import { Link } from "react-router-dom";
 import { useQuiz } from "../../Context/QuizContext";
+import { useAuth } from "../../Context/AuthContext";
 import "./Home.css";
 export const Home = () => {
   const { quiz } = useQuiz();
-
+  const {
+    state: { isAuthenticated },
+  } = useAuth();
   return (
     <div className='container'>
       <div className='bold-text'>All Categories</div>
       {quiz === undefined ? (
         <span>Loading...</span>
       ) : (
         <div className='category-container'>
-          {quiz.map((item) => (
-            <div className='category-card'>
+          {quiz.map((item, index) => (
+            <div className='category-card' key={index}>
```

Comment:
its not good to use index as  key, if you don't have id or any unique values you could use a combination of properties quiz title-index